core-aam/index.html

```diff
@@ -11199,6 +11199,132 @@ <h3>Special Events for Menus</h3>
       </section>
     </section>
 
+    <section id="mapping_algorithms">
+      <h2>Algorithms</h2>
+      <p>
+        Some <abbr title="Application Programming Interfaces">APIs</abbr> provide
+        methods which require specific algorithms to be followed. The following
+        sections provide the algorithm mapping tables for these methods.
+      </p>
+      <h3>ARIANotifyMixin Algorithm Mapping Tables</h3>
+      <p>
+        The <code>ARIANotifyMixin</code> provides methods for announcing content
+        to assistive technologies. The following algorithm mappings specify how
+        these announcements will be implemented across different accessibility
+        APIs to ensure consistent behavior for users of assistive technologies.
+      </p>
+      <h4 id="arianotifymixin-map-arianotify">ariaNotify</h4>
+      <p>To <code>aria notify</code> given <var>node</var>, <var>announcement</var>, and <var>priority</var>:</p>
+      <table aria-labelledby="arianotifymixin-map-arianotify" class="data">
+        <tbody>
+          <tr>
+            <th>ARIA Specification</th>
+            <td>
+              <a class="method-reference" href="#arianotify"><code>aria notify</code></a>
+            </td>
+          </tr>
+          <tr>
+            <th>Preconditions</th>
+            <td>
+              <p>If <var>node</var> is <a class="specref" href="#tree_exclusion">excluded from the accessibility tree</a>, then abort these steps.</p>
+            </td>
+          </tr>
+          <tr>
+            <th>Language</th>
+            <td>
+              <p>
+                User agents and assistive technologies MUST determine the language of the announcement by taking the first valid BCP&nbsp;47 language tag from the following sources:
```

Comment:
Non-blocking: Would it be useful to link to a definition of “BCP 47 language tag”?

I looked for something we could xref with respec, but https://respec.org/xref/?term=BCP+47 returns no results.